app/assets/javascripts/admin/utils/services/errors_parser.js.coffee

```diff
@@ -2,16 +2,25 @@
 angular.module("admin.utils").factory "ErrorsParser", ->
   new class ErrorsParser
     toString: (errors, defaultContent = "") =>
+      return defaultContent unless errors?
+
+      errorsString = ""
       if errors.length > 0
         # it is an array of errors
-        errorsString = error + "\n" for error in errors
+        errorsString = this.arrayToString(errors)
```

Comment:
Maybe you can use `errors.join("\n")` here instead?

Reply:
ah, yes, thanks. done :+1: